symmetric-core/src/main/resources/symmetric-default.properties

```diff
@@ -3155,7 +3163,7 @@ cloud.bulk.row.terminator=\r\n
 #
 # DatabaseOverridable: true
 # Tags: cloud, snowflake, azure
-cloud.bulk.field.quote=�
```

Comment:
Why was the cloud.bulk.field.quote parameter changed?  That seems unrelated to the issue.

Reply:
I have removed this line change.